client/src/main.tsx

```diff
@@ -2,14 +2,9 @@ import "./index.scss";
 import React from "react";
 import ReactDOM from "react-dom/client";
 import App from "./App.tsx";
-import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-
-const queryClient = new QueryClient();
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
-    <QueryClientProvider client={queryClient}>
-      <App />
-    </QueryClientProvider>
+    <App />
   </React.StrictMode>
```

Comment:
Did you mean to make this change?

Reply:
Yeah, I've removed the Query hooks so there's no need for the provider and didn't want to leave redundant code.

Reply:
but I didn't uninstall the npm package 👍